confluent_kafka/avro/load.py

```diff
@@ -27,7 +27,7 @@ def loads(schema_str):
             return schema.parse(schema_str)
         else:
             return schema.Parse(schema_str)
-    except schema.AvroException.SchemaParseException as e:
+    except schema.SchemaParseException as e:
```

Comment:
Do you mind adding a test with an invalid schema, such as the one reported in the original issue.

Reply:
Test is added and succeeds locally

Reply:
ah, flake8...hold my beer!

Reply:
fixed flake8 test and added flake8 to the dev requirements